README.md

```diff
@@ -28,16 +28,18 @@ ViteHelper\Vite\ViteHelper:
   manifestDir: '/public/manifest.json'
 ```
 
-E.g., if you use TypeScript, change the `mainJs` prop to `"main.ts"`. If you set `forceProductionMode` to true, only build files will be served - which will happen on your live site anyway. 
-Set the `devHostNeedle` to distinguish your live site from your local environment. 
+*ViteHelper Config setting options:*
+- If you set `forceProductionMode` to true, only build files will be served when running `npm run build` - which will happen on your live site anyway.
```

Comment:
-> after running the build command, not while.. 

Maybe the readme should explain Vite a bit better and that there are build files and there's a dev server.

Reply:
Yes, I agree with both.
Those who have not been exposed to Vite or modern build tools may be confused as to what Vite does and how it should be integrated.